rt/rt.py

```diff
@@ -554,16 +566,27 @@ def get_ticket(self, ticket_id: typing.Union[str, int]) -> typing.Optional[dict]
                 requestors.append(msg[req_id][12:])
                 req_id += 1
             pairs['Requestors'] = self.__normalize_list(requestors)
+            lastheader = ''
             for i in range(req_id, len(msg)):
                 if ': ' in msg[i]:
                     header, content = self.split_header(msg[i])
                     pairs[header.strip()] = content.strip()
+                    lastheader = header
+                elif len(msg[i]) == 0:
+                    # print("Empty Line")
+                    pass
+                elif msg[i] == "Cc:" or msg[i] == "AdminCc:":
```

Comment:
When does `msg[i] == "Cc:"` happen? It seems to me the line 571 picks every `msg` that has a colon. Is it true that "Cc:" cannot be multiline?